woocommerce/woocommerce
# Run daily smoke tests on wp-env

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR changes the daily smoke tests from running on an external web site to running locally on `wp-env`.  External site testing will be handled through a different workflow from now on.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to Actions > Smoke test daily
2. Start a new run using this branch.
3. Ensure that API, E2E and k6 tests run and pass.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
